test/common/secret/secret_manager_impl_test.cc

```diff
@@ -132,6 +137,51 @@ name: "abc.com"
                             "Secret type not implemented");
 }
 
+TEST_F(SecretManagerImplTest, SdsDynamicSecretUpdateSuccess) {
+  Server::MockInstance server;
+  std::unique_ptr<SecretManager> secret_manager(new SecretManagerImpl());
+
+  NiceMock<Server::Configuration::MockTransportSocketFactoryContext> secret_context;
+
+  envoy::api::v2::core::ConfigSource config_source;
+  {
```

Comment:
nit: why do you need this block?

Reply:
I think I added that block to cover some some function call, but many changes have been made and this block is not needed.